src/services/playlistManagementService.js

```diff
@@ -9,11 +9,10 @@ const managedPlaylists = require('../repositories/managedPlaylists')
 
 async function orderPlaylist (playlistId, refreshToken) {
   const spotifyAuthenticatedClient = await spotifyAuthenticationService.provideAuthenticatedClient(refreshToken)
-  let currenPlaylist = spotifyAuthenticatedClient.retrievePlaylistTracks(playlistId)
+  let currentPlaylist = spotifyAuthenticatedClient.retrievePlaylistTracks(playlistId)
   let currentTrackId = spotifyAuthenticatedClient.retrieveCurrentTrackId()
   let playlistSnapshotId = spotifyAuthenticatedClient.retrievePlaylistSnapshotId(playlistId);
-  [currenPlaylist, currentTrackId] = await Promise.all([currenPlaylist, currentTrackId]).catch((err) => { throw err })
-  const currentPlaylistTracks = currenPlaylist.items
```

Comment:
I didn't understand the removal of this step. Maybe you did it to accomodate the changes done to the unit tests, however that does not change the information retrieved via spotifyAuthenticatedClient.

Reply:
This is because of the new implementation of getting the tracks with pagination. I concatenated the playlists in `retrievePlaylistTracks` into an array with the tracks. The return of the `spotifyAuthenticatedClient` is treated in the loop that deals with the pagination. This is also why I changed the line in the previous comment.